content/module-reference/utility-modules/darkroom/color-assessment.md

```diff
@@ -24,4 +24,10 @@ Although the color assessment mode provides a mid-gray surrounding to the image,
 
 Color assessment mode can also be toggled by pressing Ctrl+B.
 
-You can modify the distance from the edge of the center panel to the edge of the image by adjusting the `darkroom/ui/iso12464_border` entry in the `$HOME/.config/darktable/darktablerc` file. In the same file you can modify the size of the white border (in percent) by adjusting the `darkroom/ui/iso12464_ratio` entry.
+The size of the grey and white border and the relative size of the white frame may be adjusted by right-clicking ![bulb-icon](./color-assessment/bulb-icon.png#icon) , which will bring up a control panel with two slider controls:
+
+- _Total border width relative to screen_: decreasing this slider will reduce the size of the combined grey and white border and will enlarge the size of the image. Increasing this control has the opposite effect.
+
+- _White border ratio_: Increasing this slider will enlarge the size of the white border and will maintain the image size, while decreasing the control will have the opposite effect.
```

Comment:
We generally use definition lists to describe module controls, and we keep the capitalisation as per the application (all lower case), so something like the following would be more consistent:

```
total border width relative to screen
: decreasing this slider will...
```